src/decision/location_selection.jl

```diff
@@ -154,15 +156,15 @@ function rank_locations(
         # Create shared decision matrix
         # Ignore locations that cannot support corals or are out of depth bounds
         # from consideration
-        decision_mat = decision_matrix(
-            dom.site_ids[valid_locs],
+        seed_decision_mat = decision_matrix(
```

Comment:
If `valid_seed_locs` is all false this will crash. Though I assume that something is wrong with the user location inputs anyway.